test/extensions/INTEL/SPV_INTEL_joint_matrix/bf16_conversion_instructions.ll

```diff
@@ -0,0 +1,72 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_KHR_cooperative_matrix,+SPV_INTEL_joint_matrix,+SPV_INTEL_bfloat16_conversion -o %t.spv
```

Comment:
Maybe add few cases where we not enable all the extensions?

Reply:
Added error check for absence of SPV_INTEL_joint_matrix. Not sure if it worth extending for other 2